zazu/repo/commands.py

```diff
@@ -81,10 +87,62 @@ def clone(ctx, repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
+@click.option('--nohooks', is_flag=True, help='does not install git hooks in the repo')
 @click.pass_context
-def init(ctx):
+def init(ctx, nohooks):
     """Initialize repo directory structure"""
-    raise NotImplementedError
+    def _zazu_yaml(issue_tracker={}, stylers=[]):
+        """builds zazu.yaml file"""
+        if issue_tracker:
+            zazu_yaml_obj = issue_tracker
+        else:
+            zazu_yaml_obj = {}
+        if stylers:
+            zazu_yaml_obj['style'] = {key: {'options': [None]} for key in stylers}
+            click.echo('Reminder: please specify styler options in  zazu.yaml')
+        yaml.dump(zazu_yaml_obj, file('zazu.yaml', 'w'), default_flow_style=False)
+    # check for git repo in cwd
+    try:
+        repo = git.Repo(os.getcwd())
+    except git.InvalidGitRepositoryError:
+        event_time = time.gmtime()
+        default_name = time.mktime(event_time)
+        # click.prompt does not play well with py format
+        repo_name = click.prompt('No existing git repo found, Name your new repo:', default='zazuRepoCreated_' + str(default_name))
+
+        try:
+            os.mkdir(repo_name)
+            repo = git.Repo.init('{}/{}/.'.format(repo_name, '.git'), bare=True)
+            os.chdir(repo_name)
+        except OSError as err:
+            raise click.ClickException(str(err))
+
+    if click.confirm("We are currently in a git repo, configure zazu.yaml?", abort=True):
+        click.echo("Configuring Zazu for: " + os.getcwd())
+        if os.path.isfile('zazu.yaml'):
+            click.confirm('zazu.yaml file found, continuing will overwrite, continue?', abort=True)
+        repo_name = os.path.basename(os.path.normpath(os.getcwd()))
+
+        click.echo('Interactive Repo Design, by zazu')
+        trackers = zazu.util.get_plugin_list(zazu.issue_tracker.IssueTracker)
+        trackers.append('None')
+        stylers = zazu.util.get_plugin_list(zazu.styler.Styler)
+        tracker_choice = zazu.util.pick(trackers, 'Pick an Issue Tracker')
+        tracker_dict = {}
+        if tracker_choice is not 'None':
+            owner = click.prompt('Please enter an owner for issues created from this repo', default=socket.getfqdn())
```

Comment:
This should be the github "owner" i.e stopthatcow or lilyrobotics we may be able to default this by asking the keychain for the github username...but perhaps we should add that functionality later.